core/src/elements/grenade.rs

```diff
@@ -209,20 +206,6 @@ fn update_lit_grenades(
             let player = inventory.player;
             let layers = player_layers.get_mut(player).unwrap();
             layers.fin_anim = *fin_anim;
-            let body = bodies.get_mut(entity).unwrap();
```

Comment:
Is this not needed for some reason?

Reply:
I believe our attachment system already handles all of these params for us